app/decorators/risk_assessment_decorator.rb

```diff
@@ -39,6 +39,13 @@ def activity_cell_partial(_viewing_user)
     "activity_table_cell_with_link"
   end
 
+  def assessed_by
+    return Business.find(assessed_by_business_id).legal_name if assessed_by_business_id
+    return Team.find(assessed_by_team_id).name if assessed_by_team_id
```

Comment:
This would potentially be more performant if the associated records are included in the search query.
```suggestion
    return assessed_by_business.legal_name if assessed_by_business_id
    return assessed_by_team.name if assessed_by_team_id
```